src/components/Footer.tsx

```diff
@@ -1,4 +1,3 @@
-/* eslint-disable react-hooks/rules-of-hooks */
```

Comment:
?

Reply:
Basically as per eslint v9.

if there is no rule define in eslint.config.ts and we are adding the command in our code. so eslint lint will show that command as warning.

so we just fix all these by running yarn lint --fix. if you want to keep these warning as it is so will revert all these.

Reply:
we have tried to remove this in your eslint v8 version so there is no any error from lint. same in v9 and when we define this command v9 and try to lint. so its just showing warning. so what should we do now ?

Reply:
I’m not sure I understand. Here’s how I would approach it:

Was this rule necessary in the old config? i.e. If it is removed from the old config, does the linter fail (as expected)? 

If so, then we need to ensure it is enforced in the new config. 

If not (unlikely) then we no longer need to disable it, and this line can be removed.